packages/manager/src/features/LoadBalancers/LoadBalancerCreate/ConfigurationDetails.tsx

```diff
@@ -0,0 +1,87 @@
+import Stack from '@mui/material/Stack';
+import { useFormikContext } from 'formik';
+import * as React from 'react';
+
+import { Autocomplete } from 'src/components/Autocomplete/Autocomplete';
+import { Box } from 'src/components/Box';
+import { TextField } from 'src/components/TextField';
+import { TooltipIcon } from 'src/components/TooltipIcon';
+import { Typography } from 'src/components/Typography';
+
+import type { CreateLoadbalancerPayload } from '@linode/api-v4';
+
+const protocolOptions = [
+  { label: 'HTTPS', value: 'https' },
+  { label: 'HTTP', value: 'http' },
+  { label: 'TCP', value: 'tcp' },
+];
+
+interface Props {
+  index: number;
+}
+
+export const ConfigurationDetails = ({ index }: Props) => {
+  const {
+    errors,
+    handleChange,
+    setFieldValue,
+    values,
+  } = useFormikContext<CreateLoadbalancerPayload>();
+
+  return (
+    <Box padding={1}>
+      <Typography variant="h2">Details</Typography>
+      <Typography sx={(theme) => ({ marginRight: theme.spacing(1) })}>
+        The port the load balancer listens on, and the protocol for routing
+        incoming traffic to the targets.
+      </Typography>
+      <Stack spacing={2}>
+        <Stack direction="row" spacing={2}>
+          <Autocomplete
+            onChange={(_, { value }) =>
+              setFieldValue(`configurations.${index}.protocol`, value)
+            }
+            textFieldProps={{
+              labelTooltipText: 'TODO',
+            }}
+            value={protocolOptions.find(
+              (option) =>
+                option.value === values.configurations?.[index]?.protocol
+            )}
+            disableClearable
+            errorText={errors[`configurations.${index}.protocol`]}
+            label="Protocol"
+            options={protocolOptions}
+          />
+          <TextField
+            errorText={errors[`configurations.${index}.port`]}
```

Comment:
I believe errors will be in this shape. Not sure why the Typescript type is wrong. Same applies to Label and Protocol

```suggestion
            errorText={errors.configurations?.[index].port}
```